### ISSUE TITLE
add a data table that permanently stores values even after bot is restarted

### ISSUE THREAD

**Issue by BlaisinX** [HardWare68/CollabDiscordBot#11]
so I could, for example, type:
`~store 0 hi`
and it would store "hi" to the first value in a data table that is stored after the bot is shut down and could be retrieved with something like:
`~value 0`

**Comment by BlaisinX** [comment]
like, web browser cookies, but for repl.it if you get what I mean

**Comment by HardWare68** [comment]
you could potentially do this with JSON. i know replit also has a built-in database but they have storage caps and all that

**Comment by BlaisinX** [comment]
something like that

**Comment by HardWare68** [comment]
yeah, we could try this with JSON. i was trying that on the old repo, so i can try and get back to it

**Comment by BlaisinX** [comment]
I have no idea how to work with JSON so i'll leave that to you

**Comment by HardWare68** [comment]
oh ok

**Comment by BlaisinX** [comment]
unless you have other things to do of course

**Comment by HardWare68** [comment]
Why yes, I do indeed have other things to do.
Your mother.

**Comment by BlaisinX** [comment]
surprisedshinx

**Comment by HardWare68** [comment]
`~do yourMom`
